.github/workflows/ci.yml

```diff
@@ -14,6 +14,15 @@ jobs:
   pest:
     name: Tests (Pest)
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        laravel: [9.*, 8.*]
+        include:
+          -   laravel: 9.*
+              testbench: 7.*
+          -   laravel: 8.*
+              testbench: ^6.9
```

Comment:
What's the `include` section for when `laravel` is already a property on `matrix`?

Reply:
It adds additional configuration options to a build matrix job that already exists. For example, when running `laravel 9.*` from the matrix, use the `testbench: 7.*`

Reply:
Right. I'm thinking that we probably don't even need these. Composer will know which `testbench` to install when installing Laravel.

Reply:
Also, you only added these values to the matrix, but you're not actually using them anywhere. The script doesn't know what `laravel` or `testbench` means.

See that the action installs the same version of Laravel in [8.x](https://github.com/archtechx/helpers/runs/4856860401?check_suite_focus=true#step:3:25) and [9.x](https://github.com/archtechx/helpers/runs/4856860459?check_suite_focus=true#step:3:25)

Reply:
> Right. I'm thinking that we probably don't even need these. Composer will know which testbench to install when installing Laravel.

Let me check if we do not install testbench, and it works fine. I'll remove the testbench command and _includes_.

Reply:
I see. We don't need to install testbench specific version because Composer does it automatically. Also, removed it from the matrix. Now matrix has just `laravel: [9, 8]`.